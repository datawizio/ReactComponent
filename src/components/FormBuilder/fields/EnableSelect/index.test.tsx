import "jsdom-global/register";
import React from "react";
import { shallow } from "enzyme";

import { FieldEnableSelect } from "./index";

const mockProps = {
  label: "label",
  name: "name",
  initialValue: "init",
  placeholder: "placeholder",
  renderField: jest.fn(),
  onChange: jest.fn()
};

const setUp = (props?) => shallow(<FieldEnableSelect {...props} />);

describe("FieldEnableSelect component", () => {
  let component;
  beforeEach(() => {
    component = setUp(mockProps);
  });

  it("FieldEnableSelect rendered correctly", () => {
    expect(component).toMatchSnapshot();
  });
});
