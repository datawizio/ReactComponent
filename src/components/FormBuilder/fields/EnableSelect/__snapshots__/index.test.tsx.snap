// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FieldEnableSelect component FieldEnableSelect rendered correctly 1`] = `
<FormItem
  className="field-enable-select"
  initialValue="init"
  label="label"
  name="name"
>
  <Field
    name="name"
    onChange={[MockFunction]}
    placeholder="placeholder"
    renderField={[MockFunction]}
  />
</FormItem>
`;
